internal/http/services/owncloud/ocs/handlers/cloud/user/user.go

```diff
@@ -109,7 +109,7 @@ func (h *Handler) getNotif(ctx context.Context) string {
 	})
 
 	if err != nil || res.Status.Code != rpc.Code_CODE_OK {
-		return ""
+		return err.Error()
```

Comment:
It would be better to just return `err` and change the return type to `(bool, error)`, you can even do the boolean parsing in this method.